README.md

```diff
@@ -80,6 +81,7 @@ Rows are listed in a row array as plain objects. Each object contains the row da
     - `row`: *(type: Object)* A vue based object for adding classes.
     - `<column-selector>`: *(type: Object)* A vue based object for adding classes. The column selector works as explained in [styling](#styling),
       without the use of slashes to distinguish columns and rows.
+- `exactMatch`: *(type: boolean, default: `false`)* Indicates row that exactly matches a filter.
```

Comment:
Please update `TableContainerApp.vue`, so that the new functionality can be seen in action, when doing `npm run serve`.
I'll continue the review from there.